utils/httpsignature/personalsign.go

```diff
@@ -0,0 +1,39 @@
+package httpsignature
+
+import (
+	"crypto"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/common"
+	ethc "github.com/ethereum/go-ethereum/crypto"
+)
+
+// EthAddress - the ethereum address hex repr
+type EthAddress string
+
+// Verify the signature sig for message based on eth address
+func (key EthAddress) Verify(message, sig []byte, opts crypto.SignerOpts) (bool, error) {
+
+	// convert key to eth address
+	address := common.HexToAddress(string(key))
+
+	// recover the public key that created the signature
+	hash := accounts.TextHash(message)
```

Comment:
the spec has now has finalized the message prefix, so we should hardcode it and include before hashing